celery_batches/__init__.py

```diff
@@ -109,6 +109,9 @@ class SimpleRequest:
 
     #: used similarly to reply_to
     correlation_id = None
+    
+    #: includes all of the original request headers
+    request_dict = {}
```

Comment:
I think this will get it to pass lint:

```suggestion

    #: includes all of the original request headers
    request_dict: Dict[str, Any] = {}
```

Reply:
(I suspect this is the last change to make, but the branch isn't writable -- do shout when you've accepted!)

Reply:
Thanks @clokep !